CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
+
+
+## [unreleased: 0.0.1]
+### Added
++ Change to include `element-session`
++ Add export notebook
+
+
+## [0.0.0b1] - 2021-03-24
+### Added
++ First beta release
+
+
+## [0.0.0a1] - 2021-03-18
+### Added
++ Added notebooks
+
+[0.0.0b1]: https://github.com/datajoint/workflow-animal/tree/f6d6a3353aae83ca13ff9fcc536017eb34c18f90
+[0.0.0a1]: https://github.com/datajoint/workflow-animal/tree/0d3d3c970056ff4c243d17cf4f738f48268d80ad
```

Comment:
- Do workflows get changelogs? I would retroactively consider the 3/18 and 3/24 changes as releases. 
- Considered present changes substantial enough to shift to 0.0.1

Reply:
We have previously implemented changelogs, tags, and releases for the `elements`, but not the `workflows`.  I would be in favor of having changelogs and tags for the `workflows`.  In which case, I would suggest keeping the versions of the `element` and corresponding `workflow` independent from each other.

Tagging for further discussion @ttngu207 @dimitri-yatsenko @iamamutt.